### DISCUSSION TITLE
Auto (BETA)

### DISCUSSION THREAD

**Discussion by OceanOv** [Rem0o/FanControl.Releases#449] [Q&A]
How to use settings "Auto (BETA)"?

**Comment by Rem0o** [comment]
![image](https://user-images.githubusercontent.com/14118956/147890718-dadfe720-bbe2-4efd-8403-feb814549595.png)

**Reply by OceanOv** [reply]
But, after all, this could be done with those tools that were previously available...

**Reply by Rem0o** [reply]
No, this will increase or reduce the speed depending if the cooling is sufficient or not.

This will try to "keep" the target load temperature. If it goes over, it will increase the speed UNTIL it goes back to the target load temperature. On the other hand, if its cooler in your room and at load it is 3-4 deg below your target load temp, the fan speed will decrease etc. Think of it like a thermostat for heating.  

Other fan curves are stateless. It's a simple formula of temperature = %. This one reads the temp continuously and adjust based on the trend of the temperature.

**Reply by koshibb** [reply]
@Rem0o - is there a way to increase the allowed temp for the auto curves?

**Reply by Rem0o** [reply]
How hot? What’s the use case ?

**Reply by koshibb** [reply]
Hey dude! You're like a movie star in  my discord! thanks for replying @Rem0o 
What I am trying to explain is that I currently am unsure of a few basic use cases of the program, and I dont want to be hindering my pc....

Here's how Id best describe them:

a) the auto curves i made for CPU, GPU and Intake fans are even the best way to go about this
b) how  I should be managing and organizing these fans either by what sensor they use or groupings. 
c) when it comes to managing fans in general, how should my bios look and what settings would i remove/add from MSI  afterburner?

thanks again dude!
